docker-compose.yml

```diff
@@ -10,22 +10,21 @@
 #                                Default: 50000
 # AIRFLOW_PROJ_DIR             - Base path to which all the files will be volumed.
 #                                Default: .
-
+---
 version: '3'
-
 x-airflow-common:
   &airflow-common
   build: .
   environment:
     &airflow-common-env
-    # Compatibility with legacy airflow.cfg
     AIRFLOW__WEBSERVER__WEB_SERVER_PORT: '8080'
     AIRFLOW__WEBSERVER__BASE_URL: 'http://localhost:8080'
     AIRFLOW__CELERY__FLOWER_PORT: '5555'
-
     AIRFLOW__CORE__EXECUTOR: CeleryExecutor
-    AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: mysql://root:secret@db:3306/airflow
-    AIRFLOW__CELERY__RESULT_BACKEND: redis://:@redis:6379/0
+    AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql+psycopg2://airflow:airflow@postgres/airflow
+    # For backward compatibility, with Airflow <2.3
+    AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql+psycopg2://airflow:airflow@postgres/airflow
+    AIRFLOW__CELERY__RESULT_BACKEND: db+postgresql://airflow:airflow@postgres/airflow
```

Comment:
just checking that you're replacing local redis with postgres

Reply:
Yeah, result backend is now postgres, but celery broker is still redis. This matches the helm chart setup